doc/mongoc_advanced_connections.page

```diff
@@ -95,6 +95,34 @@ main (int   argc,
       <p>Connecting directly to a UNIX domain socket is not a common practice.</p>
     </note>
   </section>
+   
+  <section id="SSL">
+    <info><link type="guide" xref="index#advanced-connections" /></info>
+    <title>Connecting to a server that requires SSL</title> 
+<p>These are instructions for configuring TLS/SSL connections.</p>
+   <listing> 
+   <title>Basic Configuration</title>
+    <p>To run a server locally (on port 27017, for example):</p>
+    <screen><code>$ mongod --port 27017 --sslMode requireSSL --sslPEMKeyFile server.pem --sslCAFile ca.pem </code></screen>
+    <p>If you are using a version of the C Driver earlier than 1.2.0, add <code>/?ssl=true</code> to the end of a client URI.</p>
```

Comment:
Sorry to be joining late: let's instruct users to *always* add ssl=true, without complicating the instructions by talking about the history of this behavior.